aospy/automate.py

```diff
@@ -278,7 +278,11 @@ def _compute_or_skip_on_error(calc, compute_kwargs):
 def _submit_calcs_on_client(calcs, client, func):
     """Submit calculations via dask.bag and a distributed client"""
     logging.info('Connected to client: {}'.format(client))
-    with dask.set_options(get=client.get):
+    if float(dask.__version__[:4]) < 0.18:
```

Comment:
I think xarray typically does things like this with `LooseVersion` from the standard library (e.g. [here](https://github.com/pydata/xarray/blob/cf19528d6d2baf988ad34e024cae28361c9fd693/xarray/tests/__init__.py#L91)):
```python
from distutils.version import LooseVersion

if LooseVersion(dask.__version__) < '0.18':
    dask_option_setter = dask.set_options
else:
    dask_option_setter = dask.config.set
```